samples/ar-model-viewer-compose/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.compose.compiler)
+}
+
+android {
+    namespace = "io.github.sceneview.sample.armodelviewer.compose"
+    compileSdk = 36
+    defaultConfig {
+        applicationId = "io.github.sceneview.sample.armodelviewer.compose"
+        minSdk = 28
+        targetSdk = 36
+        versionCode = 1
+        versionName = "1.0.0"
+    }
+
+    buildTypes {
+        getByName("release") {
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
+    }
+    kotlinOptions {
+        jvmTarget = "17"
+    }
+    buildFeatures {
+        compose = true
+    }
+    packagingOptions {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+    androidResources {
+        noCompress.add("filamat")
+        noCompress.add("ktx")
+    }
+}
+
+dependencies {
+    implementation(project(":samples:common"))
+
+    implementation(libs.androidx.compose.ui)
+    implementation(libs.androidx.compose.foundation)
+    implementation(libs.androidx.activity.compose)
+    implementation(libs.androidx.compose.material)
+    implementation(libs.androidx.compose.ui.tooling.preview)
+    implementation(libs.androidx.navigation.compose)
+    debugImplementation(libs.androidx.compose.ui.tooling)
```

Comment:
Many of the dependencies should be already included in the `common` module so it's possible to remove them in the samples.